.github/workflows/build-and-test-macos.yml

```diff
@@ -16,31 +16,31 @@ jobs:
       fail-fast: false
       matrix:
         include:  # Pairwise testing
-          - compiler: gcc
-            mpi: mpich
-            math-libs: openblas
-            build-shared: false
-            with-64bit-int: true
-            with-openmp: true
-            with-solver: strumpack
-            with-eigensolver: slepc
+          # - compiler: clang
+          #   mpi: openmpi
+          #   math-libs: openblas
+          #   build-shared: shared
+          #   with-64bit-int: int32
+          #   with-openmp: serial
+          #   with-solver: superlu
+          #   with-eigensolver: slepc
```

Comment:
If these are being commented out, might as well delete the lines

Reply:
This particular CI job should not be failing but for some unknown reason was. My plan is to re-enable in downstream PR to fix this issue.